.clang-format

```diff
@@ -0,0 +1,140 @@
+---
+Language:        Cpp
+# BasedOnStyle:  LLVM+WebKit+Chromium
+AccessModifierOffset: -4
+AlignAfterOpenBracket: AlwaysBreak
+AlignConsecutiveMacros: AcrossEmptyLinesAndComments
+AlignConsecutiveAssignments: AcrossEmptyLinesAndComments
```

Comment:
yes it was intentional, but we can change I don't mind anything in this case

Reply:
Not big fan of the alignment of `=` assignments neither. As of the alignment to the right, I have never seen anything like that before. I am all for a consistent formatting, really don't care too much about the rules as long as they are not too artificial,  not too far from what a human developer (think lazy) would write :grin:

Reply:
Good. I don't think I'd ever hit the TAB so many times myself :-)

So, I suggest to avoid those alignments.